supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -174,6 +175,21 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
+  private def generateDrsLocalizerManifest(inputs: List[PipelinesApiInput]): String = {
+    inputs.collect {
+      case PipelinesApiFileInput(name, drsPath: DrsPath, relativeHostPath, mount) => {
+        val drsPathStr = drsPath.pathAsString
+        val containerPathStr = PipelinesApiFileInput(name, drsPath, relativeHostPath, mount).containerPath.pathAsString
+        s"\"$drsPathStr\",\"$containerPathStr\""
+      }
+    }.mkString("\n")
+  }
```

Comment:
Just fixed a bug in the above function and it made me think this would be a good place to test.